subprojects/android/src/main/java/org/mockito/android/internal/creation/AndroidByteBuddyMockMaker.java

```diff
@@ -1,13 +1,32 @@
 package org.mockito.android.internal.creation;
 
 import org.mockito.internal.creation.bytebuddy.SubclassByteBuddyMockMaker;
+import org.mockito.internal.util.ConsoleMockitoLogger;
+import org.mockito.internal.util.Platform;
 import org.mockito.invocation.MockHandler;
 import org.mockito.mock.MockCreationSettings;
 import org.mockito.plugins.MockMaker;
 
-public class AndroidByteBuddyMockMaker implements MockMaker {
+import static org.mockito.internal.util.StringJoiner.join;
 
-    private final MockMaker delegate = new SubclassByteBuddyMockMaker(new AndroidLoadingStrategy());
+public class AndroidByteBuddyMockMaker extends ConsoleMockitoLogger implements MockMaker {
```

Comment:
I'd rather have it as a field. Actually I'm thinking about having a plugin for the logger.